packages/apollo-voyager-keycloak/src/AuthContextProvider.ts

```diff
@@ -0,0 +1,31 @@
+import { Request } from "express"
+import { IncomingMessage } from "http"
+
+export default class KeycloakAuthContextProvider {
```

Comment:
I tend to avoid `default` as it will introduce difference between typescript and js usage.
For common js default import looks like `require("x").default which is missleading

Reply:
Thanks I stopped doing it but there must be a few still lingering around :)